### PR TITLE
Update hotshot/ includes updated builder

### PR DESCRIPTION
**Pull request by move47**
Closes #<ISSUE_NUMBER>
#1415 

### This PR:

- Updates to latest builder which fixes some found issues during the staging.
- Should make the `non_permissioned_test` reliable.

### This PR does not:
changes except /builder

### Key places to review:
/env 
/builder

It should pass all the existing tests.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -98,8 +98,8 @@ ESPRESSO_BUILDER_ETH_MNEMONIC=${ESPRESSO_SEQUENCER_ETH_MNEMONIC}
 ESPRESSO_BUILDER_SERVER_PORT=41003
 ESPRESSO_BUILDER_CHANNEL_CAPACITY=1024
 ESPRESSO_BUILDER_BOOTSTRAPPED_VIEW=0
-ESPRESSO_BUILDER_WEBSERVER_RESPONSE_TIMEOUT_DURATION=1s
-ESPRESSO_BUILDER_BUFFER_VIEW_NUM_COUNT=15
+ESPRESSO_BUILDER_WEBSERVER_RESPONSE_TIMEOUT_DURATION=1500ms
```
Comment (move47) [inline .env:101]: @jbearer please check this!

Comment (jbearer) [reply .env:101]: Seems reasonable. With `ESPRESSO_ORCHESTRATOR_BUILDER_TIMEOUT=2s` this means we can take the maximum server reponse time to process a request and still have 500ms for the response to propagate to the leader before they give up

File: .gitignore [hunk 2]
```diff
@@ -26,3 +26,6 @@ node_modules/
 cache
 
 **/*.bin
+
+**.txt**
+**.json**
```
Comment (jbearer) [inline .gitignore:31]: This seems like an accident waiting to happen, can you ignore something more specific?

File: builder/src/non_permissioned.rs [hunk 3]
```diff
@@ -101,7 +101,7 @@ impl BuilderConfig {
         let (req_sender, req_receiver) = broadcast::<MessageType<SeqTypes>>(channel_capacity.get());
 
         // builder api response channel
-        let (res_sender, res_receiver) = unbounded();
+        //let (res_sender, res_receiver) = unbounded();
```
Comment (jbearer) [inline builder/src/non_permissioned.rs:104]: Delete commented out code before merging

File: builder/src/non_permissioned.rs [hunk 4]
```diff
@@ -164,9 +162,9 @@ impl BuilderConfig {
             max_api_timeout_duration,
         );
 
-        let proxy_global_api_state = Arc::new(RwLock::new(proxy_global_state));
+        //let proxy_global_api_state = Arc::new(RwLock::new(proxy_global_state));
```
Comment (jbearer) [inline builder/src/non_permissioned.rs:165]: here too

File: builder/src/permissioned.rs [hunk 5]
```diff
@@ -402,7 +402,7 @@ impl<N: network::Type, P: SequencerPersistence, Ver: StaticVersionType + 'static
         let (req_sender, req_receiver) = broadcast::<MessageType<SeqTypes>>(channel_capacity.get());
 
         // builder api response channel
-        let (res_sender, res_receiver) = unbounded();
+        //let (res_sender, res_receiver) = unbounded();
```
Comment (jbearer) [inline builder/src/permissioned.rs:405]: here too

File: builder/src/permissioned.rs [hunk 6]
```diff
@@ -477,9 +475,9 @@ impl<N: network::Type, P: SequencerPersistence, Ver: StaticVersionType + 'static
             max_api_timeout_duration,
         );
 
-        let proxy_global_api_state = Arc::new(RwLock::new(proxy_global_state));
+        //let proxy_global_api_state = Arc::new(RwLock::new(proxy_global_state));
```
Comment (jbearer) [inline builder/src/permissioned.rs:478]: here too

File: .gitignore (hunk 2 above)
Comment (move47) [reply .gitignore:31]: Oops, didn't see this.

File: builder/src/permissioned.rs (hunk 6 above)
Comment (move47) [reply builder/src/permissioned.rs]: https://github.com/EspressoSystems/espresso-sequencer/pull/1416/commits/a590a3be50ae1de9aef94e3daa11474cd8b7fa6f

File: builder/src/permissioned.rs (hunk 5 above)
Comment (move47) [reply builder/src/permissioned.rs]: https://github.com/EspressoSystems/espresso-sequencer/pull/1416/commits/a590a3be50ae1de9aef94e3daa11474cd8b7fa6f

File: builder/src/non_permissioned.rs (hunk 4 above)
Comment (move47) [reply builder/src/non_permissioned.rs]: https://github.com/EspressoSystems/espresso-sequencer/pull/1416/commits/a590a3be50ae1de9aef94e3daa11474cd8b7fa6f

File: builder/src/non_permissioned.rs (hunk 3 above)
Comment (move47) [reply builder/src/non_permissioned.rs]: https://github.com/EspressoSystems/espresso-sequencer/pull/1416/commits/a590a3be50ae1de9aef94e3daa11474cd8b7fa6f

**Review by jbearer — APPROVED** [review]

**Pull request merged**